content/sensu-enterprise/2.6/upgrading.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Upgrading Sensu Enterprise"
+weight: 14
+product: "Sensu Enterprise"
+version: "2.6"
+menu: "sensu-enterprise-2.6"
+---
+
+Upgrading Sensu Enterprise is usually a straightforward process. In most cases,
+upgrading Sensu Enterprise only requires upgrading to the
+latest package. Certain versions of Sensu Enterprise may include changes that
```

Comment:
I'd recommend using more active language in this intro, something like: "In most cases, you can upgrade Sensu Enterprise by installing the latest package."